docs/logs/getting-started/README.md

```diff
@@ -61,6 +61,12 @@ instance, which is then used to do the logging. The log is sent to the
 `OpenTelemetryLoggerProvider`, which is configured to export logs to
 `ConsoleExporter`. `ConsoleExporter` simply displays it on the console.
 
+If you are using ASP.NET Core applications, then adding logging providers
```

Comment:
I have seen users trying to create LoggerFactory by hand, and managing its lifecycle themselves (often incorrectly). Linking to the common workload type to advice users about this..

Ideally this should be covered fully in the .net logging docs, and we should simply link from opentelemetry to there. But as of today, the .NET logging docs for console apps (https://docs.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line#non-host-console-app) is very minimal..